context/build.gradle

```diff
@@ -2,5 +2,6 @@ description = 'gRPC: Context'
 
 dependencies {
     testCompile project(':grpc-testing')
+    testCompile project(':grpc-core').sourceSets.test.output
```

Comment:
I sort of wish this didn't depend on grpc-testing already... The only reason it does now is for DeadlineSubject in a single test case.